ocrd/ocrd/task_sequence.py

```diff
@@ -74,16 +83,18 @@ def __str__(self):
             ret += ' -p %s' % self.parameter_path
         return ret
 
-def validate_tasks(tasks, workspace):
+def validate_tasks(tasks, workspace, page_id=None, overwrite=False):
     report = ValidationReport()
     prev_output_file_grps = workspace.mets.file_groups
 
+    first_task = tasks.pop(0)
+    first_task.validate()
+
     # first task: check input/output file groups from METS
-    # TODO disable output_file_grps checks once CLI parameter 'overwrite' is implemented
-    WorkspaceValidator.check_file_grp(workspace, tasks[0].input_file_grps, tasks[0].output_file_grps, report)
+    WorkspaceValidator.check_file_grp(workspace, first_task.input_file_grps, first_task.output_file_grps, page_id, report)
```

Comment:
Since your `overwrite` solution is based on _removing_ existing output files right before processing (i.e. in `run_cli` / `ocrd_cli_wrap_processor`) but **after** task validation, you'll still have to exempt checking output file groups here:

```suggestion
    WorkspaceValidator.check_file_grp(workspace, first_task.input_file_grps, '' if overwrite else first_task.output_file_grps, page_id, report)
```